examples/run/alpaka/seeding_example_alpaka.cpp

```diff
@@ -101,6 +117,10 @@ int seq_run(const traccc::opts::track_seeding& seeding_opts,
     // Performance writer
     traccc::seeding_performance_writer sd_performance_writer(
         traccc::seeding_performance_writer::config{});
+    // traccc::finding_performance_writer find_performance_writer(
+    //     traccc::finding_performance_writer::config{});
+    // traccc::fitting_performance_writer fit_performance_writer(
+    //     traccc::fitting_performance_writer::config{});
```

Comment:
Just kept around as it made comparing to the CUDA easier...happy to remove.

(Same for the rest of this file etc)

Reply:
Indeed, let's not add a whole bunch of commented lines. :thinking: You should absolutely keep this code in a commit in your own repository. But I'd prefer not to put it in like this into the main repo.